### DISCUSSION TITLE
No Pair Request Dexcom One, Pixel 6 on Android 13

### DISCUSSION THREAD

**Discussion by nicolapatten** [NightscoutFoundation/xDrip#2943] [Q&A]
Hi, I've been using xdrip with my dexcom one for 3 months and it's been amazing, just changed transmitter for the first time and it's refusing to send a pair request.

All settings are correct and I am running the latest version of nightscout. Tried everything to trigger a pair request, changing the last digit and back again, restarting phone, Bluetooth on and off, old transmitter is in the microwave..!

Errors showing are 'mismatch' and 'deep sleeping errors' (was saying 'power connect errors')

Only other factors I can think of are:
. When I first tried to connect the transmitter I started sensor at the same time, realised that was wrong, stopped sensor and loaded transmitter into new sensor before trying to connect transmitter first and here we are
. I have been through an airport in the last few days with this transmitter in my hand luggage - could this have broken it in some way?

Any insight? It's been a couple of days now and getting desperate!

![inbound8515842579194142092](https://github.com/NightscoutFoundation/xDrip/assets/139696477/a28f8215-6625-45ce-9e5a-b43f7e1abd94)

**Comment by Navid200** [comment]
@jamorham I asked for this to be posted here because I was not able to trigger a pair request.

**Comment by Navid200** [comment]
@nicolapatten Please follow this to enable detail logs:
https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html

Wait for a few read cycles, may be 10-15 minutes.
Then, upload logs and post here with the identifying phase you included.
Thanks

**Reply by nicolapatten** [reply]
Thank you, I have uploaded the logs with the identifying phrase:

nicolapatten logs for GitHub discussion

**Comment by jamorham** [comment]
Double check that the transmitter id is set absolutely 100% perfectly correct.

**Comment by jamorham** [comment]
Logs indicate that you have two transmitters with the same last 2 digits of transmitter id which are within bluetooth range causing confusion. I suggest trying to move any old transmitters out of bluetooth range. You may also need to set the transmitter id to something invalid and then back again to clear out the internal state as it may have learned the wrong address and both transmitters would have the same bluetooth name.

**Reply by nicolapatten** [reply]
Yes the number is 100% correct and yes both transmitters end in J9 so the Bluetooth addresses are the same, didn't think about that potentially confusing the system as well as myself! I have put the old transmitter in the microwave because I heard that blocks the Bluetooth but will try again when I am out of the house later.

**Reply by nicolapatten** [reply]
Hi, I have now destroyed and moved out of range of my old transmitter and the system status now looks different (screenshot attached) but no pair request still. I have uploaded my log again.

![Screenshot_20230716-202257](https://github.com/NightscoutFoundation/xDrip/assets/139696477/e353531d-f209-48fd-b8a3-6ac79c3034bd)

**Reply by jamorham** [reply]
Please try the following sequence:

- Change the transmitter id to make it incorrect but with the same last 2 digits, eg just change the first digit to something else. Wait till you get a mismatch message
- Reboot your phone
- Change the transmitter id back to the correct one. 
- Wait to see if it works correctly

See if that helps clear it out and get it to pair. If if doesn't work, please upload the logs again after 30 minutes or so but I wont be able to look at those till tomorrow.

If none of these work then try saving xDrip settings out or use the cloud backup feature and then uninstall and reinstall xDrip. I apologize there isn't a cleaner way to do this but I think this is an unexpected situation we haven't observed before with two devices with the same name.

Alternatively if you don't care about any of your xDrip settings you can just uninstall and reinstall and skip all the steps above and I would expect that to work now that it is not seeing the old transmitter any more.

**Reply by nicolapatten** [reply]
Thank you, this did manage to get me a pair request, but I accepted it and sensor stopped immediately and I haven't been able to trigger another one :( I have sent my logs through again. I will try uninstalling and reinstalling tomorrow and try to save my details because I don't really want to lose all of my data if I can help it.

Thanks so much for your time on this, it's really appreciated

**Reply by Navid200** [reply]
Please explain.

You say you got a pair request and you approved.  So, why do you need another one?
If you go to Android settings, does it show the the transmitter was paired on the list?

**Reply by Navid200** [reply]
Are you saying you started the sensor again?  Are you serious?

**Reply by nicolapatten** [reply]
No I haven't started the sensor again, sorry I was getting confused with the wording! I accepted the pair request but then looked into android Bluetooth settings and it showed as a saved device but not as connected. Have just reinstalled xDrip and loaded all old settings and have had and accepted another pair request since but the same thing happened, it seems to drop straight off again

**Reply by Navid200** [reply]
That's exactly how it is supposed to be.
Please post a screenshot of the status page.  Please don't do anything before discussing it here to save time.

**Reply by nicolapatten** [reply]
Attached a screenshot of status page and android Bluetooth settings

![Screenshot_20230716-233655](https://github.com/NightscoutFoundation/xDrip/assets/139696477/a829db16-f05a-4880-a5a6-d45fd8e24e74)
![Screenshot_20230716-233704](https://github.com/NightscoutFoundation/xDrip/assets/139696477/f1b7e897-8861-4f2d-a11b-9533cc254de4)

**Reply by Navid200** [reply]
Under normal circumstances, after you approve a pair request, the red note on the status page remains until you have another read cycle.
Is the red note still showing?

**Reply by Navid200** [reply]
If you look at the logs, does it show the pairing?  Does it show unpairing after that?

**Comment by nicolapatten** [comment]
I'm not great at reading the logs but I believe this is shown at 23:51 yes. The red note at the top is not there now, I think it disappeared when I accepted the request but couldn't say for sure

![Screenshot_20230716-235316](https://github.com/NightscoutFoundation/xDrip/assets/139696477/5f057281-bbcc-455e-b92a-d57e7f617c6d)

**Reply by Navid200** [reply]
OK, the only reason I asked for looking at the logs was the red note.
The status page is very important.
Would you please show what it is showing now?

**Comment by nicolapatten** [comment]
Here is the system status page

![Screenshot_20230717-000615](https://github.com/NightscoutFoundation/xDrip/assets/139696477/748e716d-e970-4899-b68b-0ad7be927f98)

**Reply by Navid200** [reply]
That looks very good.

You can start sensor now.  Or you can go ahead and perform a restart.
If you start and it fails, it will fail in 15 minutes.  Then, you can perform the restart.  The only reason I suggest this is that I am not sure what the sequence of events has been.

These are the restart instructions:
https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

Please note the consequence of starting a sensor that has been on the body for a few days is an error in readings and the need for calibration.  All of that is explained in the instructions.
Please let me know if anything is not clear.

**Reply by nicolapatten** [reply]
That's worked!! Thank you so much both of you for your help with this, I really appreciate the time you've both taken to help me out here :)

**Reply by Navid200** [reply]
Great
